app/views/batch_contexts/_batch_context.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="row">
-  <h3 class="col bg-light">
-    <%= link_to batch_context.project_name, batch_context %> by <%= batch_context.user.email %>
-  </h3>
+  <div class="col">
+    <h1><%= batch_context.project_name %> by <%= batch_context.user.email %></h1>
+  </div>
 </div>
```

Comment:
(a) convert H3 to H1 for consistency and accessibility, (b) do not link to yourself